jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -588,8 +614,9 @@ def calculate_photometry(self, dataset=None, aperture=None, background=None,
                 background_value = (background_value * display_unit).to_value(
                     img_unit, u.spectral_density(self._cube_wave))
         else:
-            bg_reg = self.aperture._get_spatial_region(subset=background if background is not None else self.background.selected,  # noqa
-                                                       dataset=dataset if dataset is not None else self.dataset.selected)  # noqa
+            sub = background if background is not None else self.background.selected
+            dat = dataset if dataset is not None else self.dataset.selected
+            bg_reg = self.aperture._get_spatial_region(subset=sub, dataset=dat)

```

Comment:
is this change just to avoid the line overflow or is `sub` and `dat` reused elsewhere?

Reply:
just to avoid line overflow, do you want me to change it back?